openmdao/drivers/scipy_optimizer.py

```diff
@@ -534,3 +534,21 @@ def _reraise(self):
         exc = self._exc_info
         self._exc_info = None
         reraise(*exc)
+
+
+class ScipyOptimizer(ScipyOptimizerDriver):
+    """
+    Deprecated.  Use PETScKrylov.
+    """
+
+    def __init__(self, **kwargs):
+        """
+        Initialize attributes.
+
+        Parameters
+        ----------
+        kwargs : dict
+            Named args.
+        """
+        super(PetscKSP, self).__init__(**kwargs)
```

Comment:
wrong class for super

Reply:
Also need to add something like:
        warn_deprecation("'MultiFiMetaModel' component has been deprecated. Use"
                         "'MultiFiMetaModelUnStructured' instead.")
to the init.